packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -153,6 +154,62 @@ export default {
       type: Object,
       default: null,
     },
+
+    /**
+     * Whether the input allows for block quote.
+     */
+    allowBlockquote: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for bold to be introduced in the text.
+     */
+    allowBold: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for bullet list to be introduced in the text.
+    */
+    allowBulletList: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for italic to be introduced in the text.
+     */
+    allowItalic: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Whether the input allows for list item to be introduced in the text.
+     */
+    allowListItem: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
Hmm does it make sense to have this separate from "allowBulletList"? Seems like this should always be enabled if allowBulletList is enabled.

Reply:
sounds fair. I will update this